microprofile-metrics/pom.xml

```diff
@@ -37,5 +37,34 @@
             <artifactId>wildfly-arquillian-container-managed</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <version>3.0.1</version>
+            <scope>provided</scope>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/com.wadpam.openserver/jsonp -->
+        <dependency>
+            <groupId>com.wadpam.openserver</groupId>
+            <artifactId>jsonp</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
```

Comment:
why not rest-assured?

Reply:
(almost) 1:1 coverage with TT TS needs to be on the code of deployment (tested part)

Tests itself need to ensure (almost / 95%? ) the same coverage, but we went ahead with rest-assured as it's a newer and more popular approach for testing

Reply:
Fixed.